src/types/formatted/healthCareLocalFacility.ts

```diff
@@ -57,12 +64,20 @@ interface VamcFacilityServiceLocation {
   fieldApptIntroTextType?: 'remove_text' | 'customize_text' | 'use_default_text'
   /** Custom appointment intro text (if applicable). */
   fieldApptIntroTextCustom?: string
-  /** Array of additional phone numbers for appointments or contact. */
-  fieldOtherPhoneNumbers?: PhoneNumber[]
+  /**
+   * Array of additional phone numbers for appointments or contact.
+   *
+   * Comes from `location.field_other_phone_numbers`
+   */
+  appointmentPhoneNumbers?: PhoneNumber[]
+  /**
+   * Array of additional contact phone numbers.
+   *
+   * Comes from `location.field_phone`
+   */
+  contactInfoPhoneNumbers?: PhoneNumber[]
```

Comment:
I can't figure out where this `contactInfoPhoneNumbers` value is actually being defined. I see you defining the type here and then referencing it in `ServiceLocation`, but I don't see anywhere in the code where we're supplying a value to that. Same with `appointmentPhoneNumbers`. Are they coming directly from Drupal and the API client?